api/sys_monitor.go

```diff
@@ -20,7 +17,7 @@ func (c *Sys) Monitor(ctx context.Context, logLevel string) (chan string, error)
 		r.Params.Add("log_level", logLevel)
 	}
 
-	resp, err := c.c.rawRequestWithContext(ctx, r)
+	resp, err := c.c.RawRequestWithContext(ctx, r)
```

Comment:
I had to revert the changes to Monitor(ctx, logLevel) function. This function is asynchronous - it starts a go routine which escapes the scope of the function. The go routine waits for the context to be cancelled before closing. Therefore, we cannot cancel the context within the function since it would immediately cause the go routine to exit.

Thanks unit tests for catching the bug!